### DISCUSSION TITLE
策引全球投资组合：美股1号 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#154] [Announcements]
# 策引全球投资组合：美股1号 · BMPI

本文介绍策引全球投资组合美股1号的详细情况，包括交易策略、资金策略、标的池及历史回测表现等。

https://www.bmpi.dev/money/portfolios/myinvestpilot_us_1/

**Comment by madawei2699** [comment]
美股1号最近的走势又开始不错了，如何在中途介入相比组合买入已经大幅上涨的持仓标的，之前我们已经分享过一些方法，不过还有一个方法就是用[chat2invest](https://www.chat2invest.com/)来分析找可以买入的点，具体来说：

以这几个标的为例，美股1号的交易策略是吊灯止损+长期均线，一般来说长期均线远比吊灯止损低的，看图中灰色的点线就是吊灯止损的线，参数可能和组合不太一样，但是可以参考。

![](https://img.bmpi.dev/fbba7292-f41e-e25d-5b62-874564a8aa32.png)

![](https://img.bmpi.dev/9b5e577d-69a7-8ecd-6789-d3f2d1a9c176.png)

![](https://img.bmpi.dev/d2acf75a-e28c-7e4b-bdf3-d90cb31eebc8.png)

一旦价格运行到止损线以下在长期均线以上的区域，越接近长期均线，买入的价格越好，但是靠近止损线时已经具备买入条件了，也就是说可以考虑二阶段买入法，在价格下跌到止损线附近时重点关注，价格突破止损线时立即买入，如果价格运行到长期均线附近，那可以多买一些，在这两个阶段附近买入都有止损的保护，如果在止损线买入，价格突然又下跌了，那可以先止损卖出，如果在长期均线附近买入，在标的交易信号出现卖出时也可以卖出，这两个位置都属于比较安全的。

所以就算一直没有买入信号，也可以用chat2invest来分析找止损点来买入。也就是说在组合没有信号买入时，你又想买入，那可以参考这种方法，那如何开始重点关注呢？你可以通过查看一个组合是否当前回撤接近历史最大回撤时开始重点关注，并用此方法分析标的具体买入点。

美股1号历史最大回撤41%，我记得之前当前回撤出现过36%，那时候就是重点关注的时候。然后用chat2invest具体分析价格是否已经接近止损线或者长期均线了。

![](https://img.bmpi.dev/43ad5df6-016b-9f33-04f3-d32382ce3c35.png)
